scm/src/main/java/com/sandboni/core/scm/resolvers/cli/ProcessRunner.java

```diff
@@ -0,0 +1,58 @@
+
+package com.sandboni.core.scm.resolvers.cli;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+public class ProcessRunner {
+
+    private static final int DEFAULT_TIMEOUT_VALUE = 5;
```

Comment:
is 5 seconds is a reasonable timeout in case you make a big refactoring for a large application?

Reply:
Good point. I ran the command on a code base of 24.4K files, about 6M total lines, 2M Java code lines. 81 files changed. took 8.3 sec. I'll increase the timeout to 15 sec.